ibm/mas_devops/roles/suite_app_config/vars/predict.yml

```diff
@@ -5,3 +5,17 @@ mas_app_ws_kind: PredictWorkspace
 
 mas_app_cfg_delay: 120
 mas_app_cfg_retries: 30
+
+mas_app_settings_wml_instance_id: "{{ lookup('env', 'CPD_WML_INSTANCE_ID') | default('openshift', true) }}"
+mas_app_settings_wml_url: "{{ lookup('env', 'CPD_WML_URL') | default('https://internal-nginx-svc.ibm-cpd.svc:12443', true) }}"
+mas_app_settings_wml_version: "{{ lookup('env', 'WML_VERSION') | default('4.5', true) }}"
+mas_app_settings_watsonstudio_project_id: "{{ lookup('env', 'CPD_WSL_PROJECT_ID') | default('', true) }}"
```

Comment:
CPD_WSL_PROJECT_ID was removed from the one_click docs in this same PR. I think this should be CPD_PREDICT_WS_PROJECT?

Reply:
@whitfiea will fix this